File: Python/sum_of_digits.py
def main(x):
    total=0

    if x in range(1000):
        while x!=0:
            d=x%10
            total=total+d
            x=x/10
    else:
        print("Input integer is out of range.......")

    print("Sum of digits: %d" %total)

if __name__ == '__main__':
    print("Enter an integer between 0-1000")
    n=int(input())
    main(n)

File: Java/sum_of_digits.java
import java.util.*;

public class sum_of_digits{

    public static void main(String[] args) {
        int n,d,sum=0;
        System.out.println("Enter an integer between 0-1000");
        Scanner scan = new Scanner(System.in);
        n = scan.nextInt();
        scan.close();

        if (n<=1000) {
            while (n!=0) {
                d=n%10;
                sum=sum+d;
                n=n/10;
            }
        } else {
                System.out.println("Input integer is out of range.....");
        }

        System.out.println("Sum of digits: " + sum);
    }
}